src/main/java/uk/gov/hmcts/divorce/systemupdate/event/SystemRemindRespondentSolicitor.java

```diff
@@ -0,0 +1,92 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.ccd.sdk.api.callback.AboutToStartOrSubmitResponse;
+import uk.gov.hmcts.ccd.sdk.type.YesOrNo;
+import uk.gov.hmcts.divorce.citizen.notification.RespondentSolicitorReminderNotification;
+import uk.gov.hmcts.divorce.common.exception.InvalidOperationException;
+import uk.gov.hmcts.divorce.divorcecase.model.Applicant;
+import uk.gov.hmcts.divorce.divorcecase.model.Application;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+import uk.gov.hmcts.divorce.notification.NotificationDispatcher;
+
+import java.time.Clock;
+import java.time.LocalDate;
+
+import static uk.gov.hmcts.divorce.divorcecase.model.ApplicationType.SOLE_APPLICATION;
+import static uk.gov.hmcts.divorce.divorcecase.model.ServiceMethod.COURT_SERVICE;
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingAos;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SYSTEMUPDATE;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.CREATE_READ_UPDATE;
+
+@Slf4j
+@Component
+public class SystemRemindRespondentSolicitor implements CCDConfig<CaseData, State, UserRole> {
+
+    public static final String SYSTEM_REMIND_RESPONDENT_SOLICITOR_TO_RESPOND = "system-remind-respondent-solicitor-to-respond";
+
+    @Autowired
+    private RespondentSolicitorReminderNotification respondentSolicitorReminderNotification;
+
+    @Autowired
+    private NotificationDispatcher notificationDispatcher;
+
+    @Value("${respondent_solicitor.response_offset_days}")
+    private int responseReminderOffsetDays;
+
+    @Autowired
+    private Clock clock;
+
+    @Override
+    public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+
+        configBuilder
+            .event(SYSTEM_REMIND_RESPONDENT_SOLICITOR_TO_RESPOND)
+            .forState(AwaitingAos)
+            .name("Remind Respondent Solicitor")
+            .description("Remind Respondent Solicitor to respond to the application (Notice of Proceedings)")
+            .grant(CREATE_READ_UPDATE, SYSTEMUPDATE)
+            .retries(120, 120)
+            .aboutToSubmitCallback(this::aboutToSubmit);
+    }
+
+    public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(CaseDetails<CaseData, State> details,
+                                                                       CaseDetails<CaseData, State> beforeDetails) {
+
+        CaseData data = details.getData();
+
+        //to avoid incorrect event submission in case of caseworker manually submits this event
+        if (isCaseValidForThisSubmission(data)) {
```

Comment:
this event is only available to system user which is not available for caseworkers so caseworkers should not able to see this event in dropdown